package.json

```diff
@@ -34,6 +34,7 @@
     "jotai": "^2.5.0",
     "jwt-decode": "^4.0.0",
     "lottie-react": "^2.4.0",
+    "lucide-react": "^0.454.0",
```

Comment:
Why do you import this module ?

I suggest you to download package.json and package-lock.json on the testnet branch to replace the ones you have in your PR to avoid conflicts when we'll merge the PR of the contributor who is fixing starknetkit